tools/sidekick/config.json

```diff
@@ -31,6 +31,20 @@
         "**.docx**"
       ]
     },
+    {
```

Comment:
Could we have this as dropdown instead of separate button?

Reply:
<img width="351" alt="Screenshot 2023-08-16 at 1 12 39 PM" src="https://github.com/adobecom/helpx-internal/assets/118168183/bd278773-1279-4143-949c-8267b18a4f49">
Is this fine?

Reply:
The above screenshot is no longer valid. A screenshot of the latest config is now in the description